packages/wmr/src/plugins/sass-plugin.js

```diff
@@ -74,8 +82,8 @@ export default function sassPlugin({ production, sourcemap, root }) {
 		if (!path.isAbsolute(file)) {
 			file = path.join(root, file);
 		}
-		const contents = await fs.readFile(file, 'utf-8');
-		done({ file, contents });
+
+		done({ file });
```

Comment:
This doesn't solve a bug, but there is no need for us to read the file ourselves. Sass supports either `contents` or `file`, but not both. If both are set it will fallback to `contents`.